src/EditorFeatures/Core/Implementation/IntelliSense/Completion/FileSystem/FileSystemCompletionHelper.cs

```diff
@@ -71,17 +71,17 @@ public ImmutableArray<CompletionItem> GetItems(string pathSoFar, string document
 
         private CompletionItem CreateCurrentDirectoryItem()
         {
-            return new CompletionItem(_completionProvider, ".", _textChangeSpan, rules: _itemRules);
+            return CommonCompletionItem.Create(".", _textChangeSpan, rules: _itemRules);
         }
 
         private CompletionItem CreateParentDirectoryItem()
         {
-            return new CompletionItem(_completionProvider, "..", _textChangeSpan, rules: _itemRules);
+            return CommonCompletionItem.Create("..", _textChangeSpan, rules: _itemRules);
```

Comment:
Are there derived instances of CommonCompletionItem? I thought CompletionItem was just data.  If it's just data, should we only have one class for it?